### PR TITLE
chore(META): add CONTRIBUTING info with first convention rule

### PR DESCRIPTION
**Pull request by davidbarna**
@SUI-Components/developers please review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Contributing to SUI Components
+
+The following is a set of guidelines for contributing with developping new components.
+
+## Getting started
+
+### Environment
+
+To develop new packages, you only need to install Node with NPM. Fin below the default versions this repo was built with:
```
Comment (midudev) [inline CONTRIBUTING.md]: `Fin` -> `Find`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing to SUI Components
+
+The following is a set of guidelines for contributing with developping new components.
```
Comment (midudev) [inline CONTRIBUTING.md]: `developping` -> `developing`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Node: `v6+`
+* NPM: `v3`
+
+### Main commands
+
+This repo is managed with [sui-studio](https://www.npmjs.com/package/@s-ui/studio) tool.
+
+Some base commands (based on sui-studio) you should acknowledge :
+* `npm run generate <category> <name>` to create a new component.
+* `npm run co` to commit your changes with a wizard.
```
Comment (midudev) [inline CONTRIBUTING.md:19]: With a wizard?

image

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* `npm run release` publish all your changes for al packages (once your PR is merge).
```
Comment (midudev) [inline CONTRIBUTING.md]: `al` -> `all`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  onChange={onChangeHandler}
+/>
+```
+
+#### `on{SubContext}{Event}`
+For compound component, we may need to provide several callbacks.
+
+```html
+<MyDialogModal
+  onClick={onClickHandler}
```
Comment (midudev) [inline CONTRIBUTING.md:46]: If we do that, then we're assuming that the `onClick` for which handler exactly? For the submit? I don't know if maybe, if it's a component with several, we should name all of them.

File: CONTRIBUTING.md (hunk 3 above)
Comment (davidbarna) [reply CONTRIBUTING.md:19]: https://en.wikipedia.org/wiki/Wizard_(software) LOL will change.
I also mispelled the examples

Comment (midudev) [reply CONTRIBUTING.md:19]: hahahah don't worry! I know `wizard` is fine is just that I thought in Gandalf while reading it. xD

**Review by midudev — APPROVED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (davidbarna) [reply CONTRIBUTING.md:46]: I don't have a strong opinion on that. To me, it's "no subcontext for default behavior". 
In the next example, onChange is "any change" and onFilterChange would be on a specific filter...
Let's see what the rest of the guys think.

Comment (midudev) [reply CONTRIBUTING.md:46]: But `any change` is even harder to determine, isn't? Because the onFilterChange is not going to overwrite the `onChange`.

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**